changelogs/fragments/797-proxmox-kvm-cloud-init.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - proxmox_kvm - add cloud-init support (new options: ``cicustom``, ``cipassword``, ``citype``, ``ciuser``, ``ipconfig``, ``nameserver``, ``searchdomain``, ``sshkeys``) (https://github.com/ansible-collections/community.general/pull/797).
```

```suggestion
  - "proxmox_kvm - add cloud-init support (new options: ``cicustom``, ``cipassword``, ``citype``, ``ciuser``, ``ipconfig``, ``nameserver``, ``searchdomain``, ``sshkeys``) (https://github.com/ansible-collections/community.general/pull/797)."
```
(This is apparently parsed as a one-element dict, where everything before `:` is the key of the entry.)